<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string comment="windowtitle search replace">Search and Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="extensionLayout">
     <item>
      <widget class="QLabel" name="lbFilter">
       <property name="text">
        <string comment="label search file filter">Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFilter">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbDirectory">
       <property name="text">
        <string comment="label search directory">Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDirectory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDirectory">
       <property name="text">
        <string comment="button browse search directory">...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRecursive">
       <property name="text">
        <string comment="checkbox recursive search">recursive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string comment="label find">Find:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ClearLineEdit" name="leFind">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tbFindPrev">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="toolTip button search back">Search back</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/images/backward.png</normaloff>:/images/backward.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="tbFindNext">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="toolTip button search forward">Search forward</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="tbSearchAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="toolTip button search all">Search all</string>
         </property>
         <property name="text">
          <string comment="button search back">All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string comment="label replace">Replace:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ClearLineEdit" name="leReplace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QToolButton" name="tbReplace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="toolTip button replace current selection">Replace current selection</string>
         </property>
         <property name="text">
          <string comment="button replace current selection">Current</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="tbReplaceAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="toolTip button replace all">Replace all</string>
         </property>
         <property name="text">
          <string comment="toolTip button replace all">All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="leOptions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string comment="label options">Options:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tbCase">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="tooltip button case sensitive search">Case sensitive search</string>
         </property>
         <property name="text">
          <string comment="button case sensitive search">Case</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tbWrap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="tooltip button wrap search">Wrap search</string>
         </property>
         <property name="text">
          <string comment="button wrap search">Wrap</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="tbWords">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="tooltip button search words only">Search for whole words only</string>
         </property>
         <property name="text">
          <string comment="button search words only">Words Only</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="tbRegExp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="tooltip button search regexps">Search with regular expressions</string>
         </property>
         <property name="text">
          <string comment="button search regexps">RegExps</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string comment="label search domain">Domain:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QToolButton" name="tbDomain">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="tooltip button domain search">Scope of the search</string>
         </property>
         <property name="text">
          <string comment="button button domain search">Whole Text</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string comment="search results">Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbCollapseResult">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="tooltip button fold/unfold search result">Fold/Unfold results</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/images/collapse.png</normaloff>:/images/collapse.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearResult">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="tooltip button clear search result">Clear all results</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/images/trashcan_empty.png</normaloff>:/images/trashcan_empty.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="twResults">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widget/ClearLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFind</tabstop>
  <tabstop>leReplace</tabstop>
  <tabstop>tbFindPrev</tabstop>
  <tabstop>tbFindNext</tabstop>
  <tabstop>tbReplace</tabstop>
  <tabstop>tbSearchAll</tabstop>
  <tabstop>tbReplaceAll</tabstop>
  <tabstop>tbCase</tabstop>
  <tabstop>tbWrap</tabstop>
  <tabstop>tbWords</tabstop>
  <tabstop>tbRegExp</tabstop>
  <tabstop>tbDomain</tabstop>
  <tabstop>cbFilter</tabstop>
  <tabstop>cbDirectory</tabstop>
  <tabstop>tbDirectory</tabstop>
  <tabstop>cbRecursive</tabstop>
  <tabstop>tbCollapseResult</tabstop>
  <tabstop>tbClearResult</tabstop>
  <tabstop>twResults</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>leFind</sender>
   <signal>returnPressed()</signal>
   <receiver>tbSearchAll</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leReplace</sender>
   <signal>returnPressed()</signal>
   <receiver>tbReplaceAll</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbClearResult</sender>
   <signal>clicked()</signal>
   <receiver>twResults</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>722</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
